lang/en.json

```diff
@@ -263,6 +263,12 @@
           "invalidInput": "A 'defaultAccount' with value of number or string is required to update the config."
         }
       },
+      "getResolvedDefaultAccountForCWD": {
+        "errors": {
+          "invalidAccountId": "In the default override file (.hs-account), the account ID is required to be a number. Please delete the current file and generate it using `hs accounts create-override`",
```

Comment:
I will implement `uiCommandReference` and other styling with these errors, but I wanted to get feedback on the copy before I do. Will ping Jono to take a look.

Reply:
I don't think we want to reference any CLI commands from within this library. This lib should be mostly agnostic of the CLI commands b/c it can be used in other places like the VS code extension. Instead we can throw a reasonable error from here, catch it in the CLI, and then log out any sort of more-specific errors from there.

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/228/commits/2132b2299444bdcd8c31c37b4f6bd53cf28295b4